src/transformers/generation/utils.py

```diff
@@ -3012,8 +3008,8 @@ def _temporary_reorder_cache(self, past_key_values, beam_idx):
             past_key_values = self._reorder_cache(past_key_values, beam_idx)
         # Exception 2: models with different cache formats. These are limited to `DynamicCache` until their
         # cache format is standardized, to avoid adding complexity to the codebase.
-        elif "bloom" in model_class or "gptbigcode" in model_class:
-            if not isinstance(past_key_values, (DynamicCache, EncoderDecoderCache)):
```

Comment:
Is this right? I'm pretty sure the cache types still need to be a tuple

Reply:
I'm talking about the change in the isinstance call, I'm pretty sure it just takes 2 arguments

```py
In [3]: isinstance("a", str, int)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[3], line 1
----> 1 isinstance("a", str, int)

TypeError: isinstance expected 2 arguments, got 3

In [4]: isinstance("a", (str, int))
Out[4]: True
```

Reply:
oh right, missed that one!